### ISSUE TITLE
update/rework of dpm channels

### ISSUE THREAD

**Issue by Toosday** [pvme/pvme-guides#3490]
update the following channels to make them up to date and easier to find information quickly:
- #dpm-advice-faq
- #dpm-advice-mage
- #dpm-advice-melee
- #dpm-advice-range

this will also involve updating the following channels
- #mechanics
- #ability-information

for all channels this includes:
- update information for accuracy
- more concise _but still readable_ text
- **keep formatting consistent** across all channels 
    - sparing use of embeds - reserved for list information, command-type blurbs, and for grouping together video links
    - **use emphasis more in guides** - quickly find keywords in any section
    - proper use of sectioning, bullets, and spacing
- some restructuring/ordering of sections and moving information between channels to make it more appropriate
    - **avoid information duplication**
- consider more .gif examples for certain ability explanations
- **make ToC easier to use**
    - to be done last, after all other work is completed

goals:
- dpm-advice-faq should be the starting point for newbies
- guide users to find any information they require effectively
- easier to direct #pvm-help questions to specific sections/channels

**Comment by Toosday** [comment]
- made a summary of all the topics that are currently covered and in what channels to find them
- currently working on the barebones restructuring/layout of all 6 channels
- once this is done will be requesting help to gather missing information/assets

**Comment by Toosday** [comment]
more content for dpm-advice-faq
- mentions of basic pvm fallacies (armor is not a big dps upgrade once you already have some power armor, its animate dead that gives tank armor it's strength, etc.) - refer to cookie's suggestion in suggestions channel
- make a section about visual indicators perhaps (also idea by cookie)

**Comment by Toosday** [comment]
- join the thread in #editors-chat if anyone wants to keep up to date
- initial planned structure is attached for reference, its what I'll be basing the update/restructuring on
[structure.txt](https://github.com/pvme/pvme-guides/files/9374230/structure.txt)
- will give it a once-over before starting editing tonight/tomorrow to make sure I haven't missed any existing content

**Comment by Toosday** [comment]
- first draft of #dpm-advice-faq is written and attached here - [dpm-advice-faq.txt](https://github.com/pvme/pvme-guides/files/9381691/dpm-advice-faq.txt)

**Comment by Toosday** [comment]
Finished drafts:
- dpm-advice-faq - [dpm-advice-faq.txt](https://github.com/pvme/pvme-guides/files/9391467/dpm-advice-faq.txt)
- dpm-advice-mage - [dpm-advice-mage.txt](https://github.com/pvme/pvme-guides/files/9391469/dpm-advice-mage.txt)

Starting work on the next one today

**Comment by stormightpvme** [comment]
https://docs.google.com/document/d/1zOJ9xurt8QmAQaAVtvyUXdXupPFwoZDE2wwrLNZU-rw/edit?usp=sharing

Feedback document on dpm-advice-faq and dpm-advice-mage.

**Comment by Toosday** [comment]
currently working on dpm-advice-melee, will have an update on this by tomorrow

**Comment by Toosday** [comment]
Taking a break from editing for a couple days coz oof irl work is gonna be busy, so gonna leave the issue open till I get back (so that if someone wants to finish it by then they can)

Updated and revised versions of the following 3 channels:
- [dpm-advice-mage.txt](https://github.com/pvme/pvme-guides/files/9434293/dpm-advice-mage.txt)
- [dpm-advice-melee.txt](https://github.com/pvme/pvme-guides/files/9434294/dpm-advice-melee.txt)
- [dpm-advice-faq.txt](https://github.com/pv
Plus arch's initial version of the updated #ability-info channel:
- [abilitiesEmbed.txt](https://github.com/pvme/pvme-guides/files/9434298/abilitiesEmbed.txt)
me/pvme-guides/files/9434292/dpm-advice-faq.txt)

Left to do:
- #dpm-advice-ranged (rewrite whole channel)
- create new ability prioritization charts for all 3 channels (and use better colors this time)
- #ability-information should be entirely replaced with what arch shared after formatting some of the embeds, some work to do there
- #mechanics needs to be re-structured and edited to look better, be up to date, combine all the information from #ability-information that isn't currently in #mechanics and is also not already covered in one of the #dpm-advice-xxx channels (so basically re-write that channel too)

**Comment by Toosday** [comment]
okay boys we resumed work

**Comment by inferiumRS3** [comment]
Updated 4TAA stuff with new videos using ability tracker:

```
__**Method A**__
⬥ How to do it, starting with staff <:soa:869284271595069451> equipped:
    • **Step 1:** 2H auto + ability, in that order, same tick (e.g. <:AirSurge:543465115870035999> <:sonic:535533809924571136>)
    • **Step 2:** Equip DW <:praeswand:643166769518739477> <:impercore:643166751067996160> and <:gconc:869285393223254107>
    • **Step 3:** Cast a DW non-channeled ability (e.g. <:wrack:856662355952795658>)
    • **Step 4:** Wait one tick after global cooldown, then repeat from step 1
.
⬥ This means forcing an auto every 3 abilities, for example:
    • 2H <:AirSurge:543465115870035999> <:sonic:535533809924571136> → <:gconc:869285393223254107> → <:wrack:856662355952795658> → (wait 1t) → 2H <:AirSurge:543465115870035999> <:sonic:535533809924571136> → <:gconc:869285393223254107> → <:wrack:856662355952795658> → (wait 1t) → repeat

.
{
  "embed": {
    "color": 39423,
    "fields": [
      {
        "name": "__4TAA - Method A__",
        "value": "To see Method A in action, [click here](https://youtu.be/jGSsOG_z5b8)."
      }
    ]
  }
}
.embed:json

.
__**Method B**__
⬥ How to do it, starting with staff <:soa:869284271595069451> equipped:
    • **Step 1:** 2H auto + equip DW <:praeswand:643166769518739477> <:impercore:643166751067996160> + <:gconc:869285393223254107> in that order, same tick
    • **Step 2:** Cast an ability (preferably DW to benefit from crit buff)
    • **Step 3:** Cast a DW non-channeled ability (e.g. <:wrack:856662355952795658>)
    • **Step 4:** Wait one tick after global cooldown, then repeat from step 1

.
⬥ This is still forcing an auto every 3 abilities, for example:
    •  2H <:AirSurge:543465115870035999> <:gconc:869285393223254107> → <:dbreath:535533833391702017> → <:wrack:856662355952795658> → (wait 1t) → 2H <:AirSurge:543465115870035999> <:gconc:869285393223254107> → <:wrack:856662355952795658> → <:corruptblast:513190159194259467> → (wait 1t) → repeat

.
{
  "embed": {
    "color": 39423,
    "fields": [
      {
        "name": "__4TAA - Method B__",
        "value": "To see Method B in action, [click here](https://youtu.be/YOr3P54t3cM)."
      }
    ]
  }
}
.embed:json
```

**Comment by Toosday** [comment]
updated mage file with inferium's content: [dpm-advice-mage.txt](https://github.com/pvme/pvme-guides/files/9438516/dpm-advice-mage.txt)

**Comment by Toosday** [comment]
Work in progress ranged file
[dpm-advice-ranged.txt](https://github.com/pvme/pvme-guides/files/9440367/dpm-advice-ranged.txt)

**Comment by Toosday** [comment]
Sorry for the delay, my IRL has been busy for a couple days.

#ranged-dpm-advice draft will be completed and shared for feedback + review to others today.

The ability prioritization image for MELEE has been completed (pending some changes from initial feedback received, and a double-check of the numbers), MAGIC is already up-to-date I think, and I will create the one for RANGED after the draft is finished.

Will work on #mechanics after that.

**Comment by Toosday** [comment]
#ranged-dpm-advice draft has been completed and attached here - [dpm-advice-ranged.txt](https://github.com/pvme/pvme-guides/files/9463935/dpm-advice-ranged.txt)

images for melee and ranged ability prioritizations will be completed over the next couple days

working on #mechanics update now

**Comment by Toosday** [comment]
I've finished updating all 6 channels, and implementing some feedback that was given in the process. All 6 files are attached below.

I will make the pull request for them tomorrow morning, need to give it a once-over in the morning after some rest (esp. the 'Ability Prioritization' images in #dpm-advice-melee and #dpm-advice-range) to make sure I didn't make any major mistakes.

Last pending task:
- 'General Death's Swiftness Rotations' in #dpm-advice-range needs to be either filled in or the blank bits removed. Looking for feedback from rangers on this point.
- After this gets merged into pvme:master, can begin soliciting help for #3516 and add the clips as we get them.

Task should be done tomorrow :D

Files:

[mechanics.txt](https://github.com/pvme/pvme-guides/files/9485353/mechanics.txt)
[ability-information.txt](https://github.com/pvme/pvme-guides/files/9485354/ability-information.txt)
[dpm-advice-faq.txt](https://github.com/pvme/pvme-guides/files/9485355/dpm-advice-faq.txt)
[dpm-advice-mage.txt](https://github.com/pvme/pvme-guides/files/9485356/dpm-advice-mage.txt)
[dpm-advice-melee.txt](https://github.com/pvme/pvme-guides/files/9485357/dpm-advice-melee.txt)
[dpm-advice-ranged.txt](https://github.com/pvme/pvme-guides/files/9485352/dpm-advice-ranged.txt)

**Comment by Toosday** [comment]
Done :D
